README.md

```diff
@@ -1,291 +1,26 @@
 ﻿# czicompress
-[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![REUSE status](https://api.reuse.software/badge/github.com/ZEISS/czicompress)](https://api.reuse.software/info/github.com/ZEISS/czicompress)
-[![CMake](https://github.com/ZEISS/czicompress/actions/workflows/cmake.yml/badge.svg?branch=main&event=push)](https://github.com/ZEISS/czicompress/actions/workflows/cmake.yml)
-[![CodeQL](https://github.com/ZEISS/czicompress/actions/workflows/codeql.yml/badge.svg?branch=main&event=push)](https://github.com/ZEISS/czicompress/actions/workflows/codeql.yml)
 [![MegaLinter](https://github.com/ZEISS/czicompress/actions/workflows/mega-linter.yml/badge.svg?branch=main&event=push)](https://github.com/ZEISS/czicompress/actions/workflows/mega-linter.yml)
 
 Reduce the size of existing CZI files by converting them to zstd-compressed CZI files.
```

Comment:
Basically just something to have pipeline jobs retrigger (does not seem to work otherwise)
```suggestion
Reduce the size of existing CZI files by converting them to [zstd](https://github.com/facebook/zstd)-compressed CZI files.
```